applications/minotari_console_wallet/src/ui/state/app_state.rs

```diff
@@ -346,6 +347,10 @@ impl AppState {
                 .map_err(|_| UiError::PublicKeyParseError)?,
         };
         let output_features = OutputFeatures { ..Default::default() };
+        let payment_id_u64: u64 = payment_id_hex
+            .parse::<u64>()
+            .map_err(|_| UiError::HexError("Could not convert payment_id to bytes".to_string()))?;
+        let payment_id = PaymentId::U64(payment_id_u64);
```

Comment:
We should allow 256 bytes, not only 8.

Reply:
This is just UI support for now, in CW to only support u64